lib/fluent/plugin/in_syslog.rb

```diff
@@ -75,6 +75,8 @@ class SyslogInput < Input
     config_param :tag, :string
     desc 'The transport protocol used to receive logs.(udp, tcp)'
     config_param :protocol_type, :enum, list: [:tcp, :udp], default: :udp
+    desc 'The message frame type.(normal, octet_count)'
+    config_param :frame_type, :enum, list: [:normal, :octet_count], default: :normal
```

Comment:
I want better name for `normal`.

Reply:
I think so, too.
How about `older` or `traditional`?

Reply:
`traditional` seems good. Will change soon.